tests/acceptance/features/webUIUpload/upload.feature

```diff
@@ -28,6 +28,15 @@ Feature: File Upload
       | new-lorem.txt                |
       | lorem.txt                    |
 
+  @smokeTest
+  Scenario: simple upload of a folder that does not exist before
+    When the user uploads folder "/home/seluser/filesForUpload" using the webUI
```

Comment:
Does this works running outside of drone? What kinds of setup steps are needed for this?

Reply:
It does now. Just set `LOCAL_UPLOAD_DIR` env variable. And, it should fly.